zerver/templatetags/app_filters.py

```diff
@@ -82,22 +83,28 @@ def render_markdown_path(markdown_file_path, context=None):
     if context is None:
         context = {}
 
+    jinja = engines['Jinja2']
+    search_path = os.path.join(settings.STATIC_ROOT,
+                               'generated', 'bots')
+    if search_path not in jinja.env.loader.searchpath:
+        jinja.env.loader.searchpath.append(search_path)
```

Comment:
Why can't this logic just be done inside `zproject/settings.py` where we set the search path in the first place?